ncm-iptables/src/main/perl/iptables.pm

```diff
@@ -343,47 +345,47 @@ sub GetPathEntries {
     my $entries = {};
 
     # Check the input parameters.
-    if ( ! defined $path   || $path   eq "" ) {
-	$? = 1;
-	$@ = "resource path to query empty";
-	return $entries;
+    if (!defined $path || $path   eq "") {
+        $? = 1;
+        $@ = "resource path to query empty";
```

Comment:
wherever this used / catched, please make sure it's locallized, see https://github.com/quattor/CAF/pull/128